dojo/db_migrations/0062_add_vuln_id_from_tool.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 2.2.16 on 2020-11-04 16:56
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('dojo', '0061_jira_webhook_secret'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='finding',
+            name='vuln_id_from_tool',
+            field=models.CharField(blank=True, help_text='Non-unique technical id from the source tool associated with the vulnerability type.', max_length=500, null=True, verbose_name='Vulnerability ID from tool'),
```

Comment:
Should be TextField eventually, to allow for more than 255 chars.

Reply:
no please. it's just an id, maybe 64 or 128.